specviz/widgets/plotting.py

```diff
@@ -168,7 +168,10 @@ def __init__(self, title=None, model=None, visible=True, *args, **kwargs):
         self._region_text_item = pg.TextItem(color="k")
         self.addItem(self._region_text_item, ignoreBounds=True)
         self._region_text_item.setParentItem(self.getViewBox())
+
+        # Removes ability to automatically change units to match science notation
         self.getAxis('bottom').enableAutoSIPrefix(False)
+        self.getAxis('left').enableAutoSIPrefix(False)
```

Comment:
Is this a fix for https://github.com/spacetelescope/specviz/issues/522 ?  If so, then probably best to add mJy and uJy into the list units one can convert to for the y-axis units.

Reply:
It is. Like I said before, I can set `include_prefix_units=True` but the result is the picture I showed before.